apps/common/src/python/mediawords/languages/ru/ru_stop_words.txt

```diff
@@ -16,894 +21,691 @@ rnd
 sid
 style
 www
-а
 А
-августа
-акций
-Александр
-Александра
-Алексей
-Анатолий
-Андрей
+Б
+В
+Г
+Д
+Е
+И
+К
+Л
+М
+Н
+О
+П
+С
+У
+Ю
+Я
+а
+в
+г
+д
+е
+ж
+и
+й
+к
+м
+о
+с
+т
+у
+х
+я
+с кем
 АО
+Во
+Вы
+Да
```

Comment:
Could you sort this list and others, also remove duplicates if there are any? At least Sublime Text has multilanguage-aware sorting functionality (Edit -> Sort Lines and Edit -> Permute Lines -> Unique might come in handy).

Reply:
Somehow the thing still didn't get sorted :( Could you resort other lists too?